store/src/java/com/zimbra/qa/unittest/TestMailItem.java

```diff
@@ -34,29 +39,47 @@
 import com.zimbra.cs.mailbox.MailboxManager;
 import com.zimbra.cs.service.formatter.HeadersOnlyInputStream;
 
-public class TestMailItem extends TestCase {
+public class TestMailItem {
 
-    private static final String USER_NAME = "user1";
-    private static final int TEST_CONTACT_ID = 9999;
+    @Rule
+    public TestName testInfo = new TestName();
+    private String USER_NAME = null;
 
-    @Override
+    @Before
     public void setUp() throws Exception {
-        cleanUp();
+        USER_NAME = "testmailitem-" + testInfo.getMethodName() + "-user";
+        tearDown();
     }
 
+    @After
+    public void tearDown() throws Exception {
+        TestUtil.deleteAccountIfExists(USER_NAME);
+    }
+
+    @Test
     public void testListItemIds()
     throws Exception {
-        Account account = TestUtil.getAccount(USER_NAME);
+        Account account = TestUtil.createAccount(USER_NAME);
+        // Mailbox.ID_FOLDER_INBOX;
         Mailbox mbox = MailboxManager.getInstance().getMailboxByAccount(account);
+        TestUtil.addMessage(mbox, testInfo.getMethodName() + "missive 1");
```

Comment:
nit: since user name is already unique for this test method, there is no need to use test method in message body or subj